src/internal/user_api.rs

```diff
@@ -312,7 +312,7 @@ impl Jwt {
         // This `dangerous_unsafe_decode` is safe because the server will do the actual
         // signature verification and validation. We just want to do a little initial validation
         // to catch issues earlier.
-        let token_data = jsonwebtoken::dangerous_unsafe_decode::<JwtClaims>(jwt)
+        let token_data = jsonwebtoken::dangerous_insecure_decode::<JwtClaims>(jwt)
```

Comment:
Oh hey, I'm the one who renamed that function in `jsonwebtoken` and deprecated the old name. Guess I never remembered to use the new name here.

Reply:
I thought that was you :)